### DISCUSSION TITLE
Is there an option to specify which link is downloaded/played for a given episode?

### DISCUSSION THREAD

**Discussion by MysteriousLounger** [pystardust/ani-cli#1450] [Q&A]
I vaguely remember that there used to be an option to select which provider would be used to download an episode, but that was a good while ago, and the current help page doesn't list any similar option to force a certain link. Anyways, I encountered an episode (Ranma 1/2 ep 4) that had no subtitles after I downloaded it. I checked on allanime, and indeed some of the sources have no subtitles, while others do have the correct subtitles. Is there an option to make the program download from a particular link, so that I can manually tell it to choose a source that has the correct subtitles burned in?

**Comment by port19x** [comment]
No, but you can select the video quality, which may default to a different provider
